include/gz/sensors/EnvironmentalData.hh

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
Should we follow this up wit a pr to gz-sim7 removing the corresponding EnvironmentalData.hh files there and simply including the files here so that we have on source of ground truth?

Reply:
@arjo129 yes! That's what https://github.com/gazebosim/gz-sim/pull/1804 does.